aerogear/unifiedpush-operator
# ✨ Add optional DatabaseSecret field to CRD

JIRA: https://issues.redhat.com/browse/INTLY-4379

I have deployed this on my cluster for verification, in three different configurations:

- Default, with operator-provisioned postgresql
- Pre-existing external db flow, where the details are provided through the CR
- New external db flow, where the db details are provided through a pre-existing secret

These are deployed to the following namespaces respectively:
- intly-4379a
- intly-4379b
- intly-4379c

See the JIRA issue or contact me for details of the cluster.

Verification steps:
- Make sure that everything is in a ready state & UPS can be accessed.
- Maybe try creating a Push Application
- Make sure there are no persistent errors in the unifiedpush-operator pod (those "unable to decode an Event" log messages caused by enmasse don't count -- we don't use that currently anyway)
- Take a look at the UnifiedPushServer CR, and notice that the `status.secondaryResources` is different (and reasonable) in each scenario
